Tests/SuperLinq.Test/BatchTest.cs

```diff
@@ -9,11 +9,11 @@ public void BatchIsLazy()
 		_ = new BreakingSequence<int>().Buffer(1);
 
 		_ = new BreakingSequence<int>()
-			.Batch(1, BreakingFunc.Of<ArraySegment<int>, int>());
+			.Batch(1, BreakingFunc.OfSpan<int, int>());
 		_ = new BreakingSequence<int>()
-			.Batch(new int[2], BreakingFunc.Of<ArraySegment<int>, int>());
+			.Batch(new int[2], BreakingFunc.OfSpan<int, int>());
 		_ = new BreakingSequence<int>()
-			.Batch(new int[2], 1, BreakingFunc.Of<ArraySegment<int>, int>());
+			.Batch(new int[2], 1, BreakingFunc.OfSpan<int, int>());
 	}
 
 	[Fact]
```

Comment:
@viceroypenguin In `BatchTests`:

- `BatchValidatesSize` validates the exceptions thrown when `size` is invalid. Should we do the same for the other arguments?